app/views/providers/means_reports/show.html.erb

```diff
@@ -73,4 +73,10 @@
 
   <%= render 'shared/check_answers/assets', read_only: true %>
 
+  <% if @legal_aid_application.passported? && @cfe_result.capital_contribution_required? %>
```

Comment:
Is this just for passported applications?

Reply:
Yes, the non-passported caseworker review section is handled further up in a different sequence